src/main/resources/api-definition.yaml

```diff
@@ -177,6 +254,74 @@ resourceTypes:
 
 /api:
   /requisitions:
+      put:
+          is: [ secured ]
+          description: When saveAll param is present, it will save all requisitions from the request body
+          queryParameters:
+              saveAll:
+                  displayName: saveAll
+                  type: string
+                  required: false
+                  repeat: false
+          body:
+              application/json:
+                  schema: approveRequisitionDtoArray
+          responses:
+              "200":
+                  body:
+                    application/json:
+                        schema: requisitionsProcessingStatus
+              "400":
+                  body:
+                    application/json:
+                        schema: requisitionsProcessingStatus
+      post:
+          is: [ secured ]
+          description: when approveAll param is present, it will approve requisitions with passed ids
+          queryParameters:
+              approveAll:
+                  displayName: approveAll
+                  type: string
+                  required: false
+                  repeat: false
+              id:
+                  displayName: id
+                  type: string
+                  required: false
+                  repeat: true
+          responses:
+              "200":
+                  body:
+                    application/json:
+                        schema: requisitionsProcessingStatus
+              "400":
+                  body:
+                    application/json:
+                        schema: requisitionsProcessingStatus
+      get:
+          is: [ secured ]
+          description: when retrieveAll param is present, it will retrieve requisitions with passed ids
+          queryParameters:
+              retrieveAll:
+                  displayName: retrieveAll
+                  type: string
+                  required: false
+                  repeat: false
+              id:
+                  displayName: id
+                  type: string
+                  required: false
+                  repeat: true
+          responses:
+              "200":
+                  body:
+                    application/json:
+                        schema: requisitionsProcessingStatus
+              "400":
```

Comment:
maybe I'm missing something, can you point me to place in retrieveAll endpoint where it return 400?